Desktop/html/js/analysis.js

```diff
@@ -126,8 +126,8 @@ JASPWidgets.AnalysisView = JASPWidgets.View.extend({
 
 	_setTitle: function (title, format) {
 
-		this.viewNotes.firstNoteNoteBox.ghostText = title + ' - Introduction: ' + this.viewNotes.firstNoteNoteBox.ghostTextDefault;
-		this.viewNotes.lastNoteNoteBox.ghostText  = title + ' - Conclusion: '   + this.viewNotes.lastNoteNoteBox.ghostTextDefault;
+		this.viewNotes.firstNoteNoteBox.ghostText = title + ` - ${i18n("Introduction")}: ` + this.viewNotes.firstNoteNoteBox.ghostTextDefault;
+		this.viewNotes.lastNoteNoteBox.ghostText  = title + ` - ${i18n("Conclusion")}: `   + this.viewNotes.lastNoteNoteBox.ghostTextDefault;
```

Comment:
Only `Introduction` and `Conclusion` is translated, but you should add the semicolon: in French you should normally add a space before the semicolon, so it should be translated by `Introduction :`

Reply:
>Only Introduction and Conclusion is translated

that's OK ,other variables (such `title`) are translated some where by `i18n(...)` call or from QML/C++ level, because we just translate strings Literal.

> add the semicolon

I just let `:`including translation string, so that translator can change the style for they locale?